db/migrate/20160831100250_add_field_type_to_custom_attributes.rb

```diff
@@ -0,0 +1,5 @@
+class AddFieldTypeToCustomAttributes < ActiveRecord::Migration[5.0]
+  def change
+    add_column :custom_attributes, :field_type, :string, :default => "string"
```

Comment:
Please remove the default value from the database and use default_value_for in the model.  cc @gtanzillo (for future reference)